src/gpl/src/AbstractGraphics.cpp

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2020-2025, The OpenROAD Authors
+
+#include "gpl/AbstractGraphics.h"
+
+namespace gpl {
+
+AbstractGraphics::~AbstractGraphics() = default;
```

Comment:
Why not put this in the header?

Reply:
There are likely many interfaces that don't follow this rule.  Since we don't use shared libraries it has never mattered but it could in the future so it is a good idea.